subsys/mgmt/mcumgr/grp/os_mgmt/src/os_mgmt_reboot_bt.c

```diff
@@ -35,7 +35,7 @@ static void bt_disable_work_handler(struct k_work *work)
 		LOG_ERR("BT disable failed before reboot: %d", err_rc);
 	}
 #endif
-#ifdef CONFIG_MPSL
+#if defined(CONFIG_MPSL) && !defined(CONFIG_BT_UNINIT_MPSL_ON_DISABLE)
```

Comment:
no fixup commits, fix in original commit

Reply:
moved to initial commit.